draft-ietf-tls-tls13.md

```diff
@@ -2757,10 +2758,16 @@ Structure of this message:
        opaque DistinguishedName<1..2^16-1>;
 
        struct {
+           opaque certificate_extension_oid<0..2^8-1>;
+           opaque certificate_extension_values<0..2^8-1>;
```

Comment:
Yes, will fix.

From: ekr [mailto:notifications@github.com]
Sent: Tuesday, August 18, 2015 7:54 AM
To: tlswg/tls13-spec <tls13-spec@noreply.github.com>
Cc: Andrei Popov <Andrei.Popov@microsoft.com>
Subject: Re: [tls13-spec] TLS Client Auth (#209)


In draft-ietf-tls-tls13.md<https://github.com/tlswg/tls13-spec/pull/209#discussion_r37307760>:

> @@ -2757,10 +2758,16 @@ Structure of this message:

>         opaque DistinguishedName<1..2^16-1>;

>

>         struct {

> +           opaque certificate_extension_oid<0..2^8-1>;

> +           opaque certificate_extension_values<0..2^8-1>;

oid seems like it should be non-empty.

—
Reply to this email directly or view it on GitHub<https://github.com/tlswg/tls13-spec/pull/209/files#r37307760>.